.github/workflows/code-health.yml

```diff
@@ -28,7 +23,39 @@ jobs:
           node-version: 16.x
       - name: Install dependencies
         run: yarn install --check-files
-      - name: build
-        run: npx projen build
-
+      - name: compile
+        run: npx projen compile
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color
+        shell: bash
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
```

Comment:
any reason to use 16, this version is going EOL this year and people should start migrating to 18

Reply:
oh sorry I missed this. I will update it to see if we can run it with 18. Thanks!

Reply:
updated